docs/templates/index.md

```diff
@@ -69,14 +69,37 @@ coder templates create <template-name>
 > [examples/](https://github.com/coder/coder/tree/main/examples/templates)
 > directory in the repo.
 
-## Configure Max Workspace Autostop
+## Template policies (enterprise)
 
-To control cost, specify a maximum time to live flag for a template in hours or
-minutes.
+By default, developers have unlimited workspace usage and can use any template version. Template policies allows you to:
```

Comment:
Should add something about how developers have full control of their workspace schedules without policy. The policy is just the enforcement layer. It reads to me like the community edition doesn't have scheduling at all.

Reply:
Yeah, I agree. I will rephrase